index.css

```diff
@@ -0,0 +1,85 @@
+@import url("https://fonts.googleapis.com/css2?family=Fruktur&display=swap");
+
+:root {
+  --regal-blue: #2a2840;
+  --regal-blue-dark: #000022;
+  --regal-blue-transparent: #2a284083;
+  --fruktur-font: "Fruktur", cursive;
```

Comment:
Better to use a generic variable name for font so that in case you change the font it makes more sense

Reply:
okay